woodwork/tests/accessor/test_column_accessor.py

```diff
@@ -344,7 +344,7 @@ def test_remove_standard_semantic_tag(sample_series):
     series = convert_series(sample_series, Categorical)
     # Check that warning is raised if use_standard_tags is True - tag should be removed
     series.ww.init(logical_type=Categorical, semantic_tags='tag1', use_standard_tags=True)
-    expected_message = "Removing standard semantic tag(s) 'category' from column 'sample_series'"
+    expected_message = 'Standard tags are used by the data frame and have been added to "sample_series"'
```

Comment:
This is not the right message for this scenario. We are are removing a standard tag (`category`) from the series, and we want to warn the user that this has been done. Also, we are doing this on a Series but the message references a dataframe.

Reply:
✅ 9bce6ab